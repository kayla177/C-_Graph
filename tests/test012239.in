ENTITY 011EEAIM name_011E journal
ENTITY 5ORJF6EN name_5ORJ conference
ENTITY 95X6HTOR name_95X6 paper
ENTITY 6FQ7RJ5I name_6FQ7 conference
ENTITY KJCL5M5U name_KJCL conference
ENTITY IKLPUB76 name_IKLP institution
ENTITY LD1SJ7IO name_LD1S paper
ENTITY HHSB604F name_HHSB journal
ENTITY TMY3Q1JQ name_TMY3 journal
ENTITY R8JJNBRW name_R8JJ institution
RELATIONSHIP TMY3Q1JQ authored_by KJCL5M5U 7.9
RELATIONSHIP KJCL5M5U collaborated_with IKLPUB76 4.9
RELATIONSHIP 011EEAIM authored_by R8JJNBRW 8.5
RELATIONSHIP R8JJNBRW affiliated_with LD1SJ7IO 6.7
RELATIONSHIP 95X6HTOR cited_by HHSB604F 5.5
RELATIONSHIP 5ORJF6EN cited_by TMY3Q1JQ 5.3
RELATIONSHIP IKLPUB76 cited_by 95X6HTOR 9.0
RELATIONSHIP R8JJNBRW presented_at 95X6HTOR 1.1
RELATIONSHIP LD1SJ7IO presented_at IKLPUB76 8.6
RELATIONSHIP 5ORJF6EN cited_by LD1SJ7IO 8.9
RELATIONSHIP R8JJNBRW affiliated_with 5ORJF6EN 0.2
RELATIONSHIP LD1SJ7IO presented_at TMY3Q1JQ 9.9
RELATIONSHIP LD1SJ7IO affiliated_with 6FQ7RJ5I 6.9
RELATIONSHIP 6FQ7RJ5I cited_by 95X6HTOR 8.6
RELATIONSHIP TMY3Q1JQ published_in IKLPUB76 1.2
HIGHEST
PRINT IKLPUB76
DELETE TMY3Q1JQ
PRINT LD1SJ7IO
HIGHEST
DELETE HHSB604F
DELETE 011EEAIM
DELETE IKLPUB76
HIGHEST
DELETE KJCL5M5U
EXIT
