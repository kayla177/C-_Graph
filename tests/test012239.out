success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
011EEAIM HHSB604F 38.3
KJCL5M5U 95X6HTOR LD1SJ7IO TMY3Q1JQ
success
R8JJNBRW IKLPUB76 5ORJF6EN 6FQ7RJ5I
011EEAIM HHSB604F 38.3
success
success
success
5ORJF6EN 95X6HTOR 24.4
success
